content/support.rst

```diff
@@ -0,0 +1,212 @@
+phone call.
+
+Support requests can be logged by partners, customers or the Support team.
+
+When a support request is logged by a customer or partner, an email is sent to
+the support team.
+
+When one of the members of the support team logs a support request, he/she
+should consider obtaining and collating as much information as possible,
+ensuring that any of the support team members can pick up the SR at any time
+without recourse to the owner.
+
+All files related to a support request should also be available in the SR
+system.
+
+
+Initial Assessment
+------------------
+
+When a new support request is logged it should receive a response within the
+same or next business day.
+
+The SR will be allocated to an Owner, who should do the initial assessment:
+
+* Verify all details logged against the call, checking what additional
+  information is required;
+
+* Make initial contact with the originator, acknowledge that we have receive
+  the request;
+
+* Verify understanding of initial information provided;
+
+* Ask for additional information if required and/or provided initial diagnosis;
+
+* If more time is needed before providing an answer, inform about the
+  approximate time.
+
+If a customer asks for multiple support questions in a single email, it is
+better to respond in multiple emails, creating a new ticket for each email.
+
+
+Managing a Support Request 
+--------------------------
+
+Once a ticket has been responded, close the ticket (a notification will not be
+sent to the customer).
+
+The ticket will re-open when the customer responds.
+
+Do not close the ticket if the response is a 'holding' type of response.
+
+
+Escalating a Support Request
+----------------------------
+
+If the owner of a SR is not be able to answer the issue raised by the customer
+and they have gone through the additional steps to obtain more information,
+they can discuss the issue within the Support team.
+
+This is especially the case if the issue is on legacy software, customized
+software (ie a new module was created for the customer only), or it is a
+potential defect in the software.
+
+The ticket holder is still the owner.
+
+When escalating an SR to the development team, a new Trac ticket should be
+created with details of the customer and link to the helpdesk conversation.
+
+Each support request should have an unique ID which will be the Trac ticket ID
+associated with this support case.
+
+Depending on the urgency of the SR, the development team should address the
+issue, investigate and provide feedback/fix.
+
+If a defect is found, create a ticket in Trac with priority High and notify the
+customer.
+
+
+Holding a screenshare session
+-----------------------------
+
+GoToMeeting can be used to conduct a screenshare or meeting session with the
+customer if the issue cannot be solved via email.
+
+Please contact the team to create a schedule or for access to GoToMeeting.
```

Comment:
I don't understand this. Support team members should have full access to GoToMeeting and create their own rooms or meetings